docs/saml.md

```diff
@@ -46,14 +54,39 @@ To authenticate with SAML rather than the basic auth:
 1. Enable SAML on the "client" side.
     1. Make sure your `.env` has the `SAML_*` defaults from `.env.example`.
     1. In your `.env`, set `BASIC_ENABLED=` and `SAML_ENABLED=1`.
-1. In another terminal, start the server (or restart, if already running).
+    1. If your `.env`'s `API_REDIRECT` looks like `http://web:8080`, you'll need to update `SAML_CONSUMER_SERVICE_URL` to use the same alias, eg `http://web:3000/auth/saml/callback`
 
-    ```shell
-    make run
-    ```
+### Optional configutation changes
+
+Since wso2is is very verbose, you may wish to reduce its logging levels. You can adjust this under Configurations (menu at far left) > Logging > Global Log4j Configuration. These changes will take effect immediately, though be warned that the server will always log the first 70ish INFO statements during startup.
+
+## Running the full eApp cluster with SAML
+
+Once you have completed the initial configuration, you can launch the full cluster including wso2is:
+
+```shell
+make run-saml
+```
+
+Or if you still have `make identity` running, you can launch the rest of the cluster with:
+
+```shell
+make run
+```
+
+Sometimes when cancelling the process, docker fails to shut down the containers. You can drop them with:
+
+```shell
+make stop-saml
+```
+
+### Login with PIV or CAC
```

Comment:
Originally there was simultaneously support for simple authentication via a form as well as with a SAML provider. The button used the "Login with PIV/CAC" label, but really it meant "Login with SAML". Now the button should just say "Login". No PIVs or CACs are involved.